tests/storage/nbd_test.py

```diff
@@ -132,6 +132,7 @@ def nbd_env(monkeypatch):
 
 @broken_on_ci
 @requires_privileges
+@pytest.mark.root
```

Comment:
These tests do not need root so they should not be marked for "root". They must run as non privileged user, and access systemd via supervdsm.

Please remove the marker from this module.

Reply:
That's why I put them in an isolated commit, I could smell that this may not be correct.

I understand the explanation, but then they are picked by the "not root" job, which will skip them because vdsm is not a privileged user. No difference, as these are broken in the ci, but that's the reasoning of the change.

I'll remove the commit though.